examples/renderers/DIB-R/.gitignore

```diff
@@ -0,0 +1 @@
+results/
```

Comment:
Not a big fan of multiple .gitignore can you put this in the root .gitignore ?

Reply:
Done.